bug #93679

shankaragoudab opened the pull request:
SPV > Multiple emails getting triggered on Organisation and Domain rejection without the reason SPV admin have added
1. added email body property in application.properties
2. Made the code changes in notification service impl

shankaragoudab committed:
bug #93679
SPV > Multiple emails getting triggered on Organisation and Domain rejection without the reason SPV admin have added
1. added email body property in application.properties
2. Made the code changes in notification service impl

shankaragoudab committed:
bug #93679
SPV > Multiple emails getting triggered on Organisation and Domain rejection without the reason SPV admin have added
1. added email body property in application.properties
2. Made changes in NotificationServiceImpl

shankaragoudab committed:
bug #93679
SPV > Multiple emails getting triggered on Organisation and Domain rejection without the reason SPV admin have added
1. added email body property in application.properties
2. Made changes in NotificationServiceImpl

karthik-tarento commented on src/main/resources/application.properties:
```diff
@@ -81,6 +81,8 @@ domain.host=http://localhost
 mdo.approval.base.url=/app/approval/{id}/to-approve
 mdo.email.template=mdoMailTemplate
 
+notification.email.body=Your request to update #fieldKey to #toValue is #state ,
```
Where this property is used ??

shankaragoudab replied:
reverted unused property.

shankaragoudab committed:
bug #93679
SPV > Multiple emails getting triggered on Organisation and Domain rejection without the reason SPV admin have added
1 reverted unused property.
